src/spendpolicy.rs

```diff
@@ -285,10 +440,7 @@ impl fmt::Display for SpendPolicy {
         match self {
             SpendPolicy::Above(height) => write!(f, "above({})", height),
             SpendPolicy::After(time) => {
-                let duration = time
-                    .duration_since(time::UNIX_EPOCH)
-                    .map_err(|_| fmt::Error)?;
-                write!(f, "after({})", duration.as_secs())
+                write!(f, "after({})", time.unix_timestamp())
```

Comment:
I think we might still be missing test for these. At least I can't seem to find the string `above(` or `after(` in any tests.

Reply:
I'm tempted to remove the display impl so I don't have to write a test for something we're never going to use.

Reply:
Fair enough